src/nvim/api/private/helpers.c

```diff
@@ -469,23 +472,33 @@ static inline void typval_encode_dict_end(EncodedData *const edata)
 #endif
 }
 
-#define TYPVAL_ENCODE_CONV_DICT_END() \
+#define TYPVAL_ENCODE_CONV_DICT_END(tv, dict) \
     typval_encode_dict_end(edata)
 
 #define TYPVAL_ENCODE_CONV_RECURSE(val, conv_type) \
     TYPVAL_ENCODE_CONV_NIL()
 
-// object_convert_one_value()
+#define TYPVAL_ENCODE_SCOPE static
+#define TYPVAL_ENCODE_NAME object
+#define TYPVAL_ENCODE_FIRST_ARG_TYPE EncodedData *const
+#define TYPVAL_ENCODE_FIRST_ARG_NAME edata
+// _object_convert_one_value()
 // encode_vim_to_object()
-TYPVAL_ENCODE_DEFINE_CONV_FUNCTIONS(static, object, EncodedData *const, edata)
```

Comment:
the `_object_convert_one_value` and `encode_vim_to_object ()` comments where to enable "grepping" for the result of `TYPVAL_ENCODE_DEFINE_CONV_FUNCTIONS`. I guess those comments can be removed now?

Same for the other case in `eval.c`.